package.json

```diff
@@ -29,17 +30,36 @@
 	},
 	"devDependencies": {
 		"@babel/core": "^7.9.0",
+		"@testing-library/react-native": "^7.2.0",
+		"@types/jest": "^27.0.2",
 		"@types/react": "~16.9.35",
 		"@types/react-native": "^0.65.0",
+		"@types/react-test-renderer": "^17.0.1",
 		"@typescript-eslint/eslint-plugin": "^4.31.0",
 		"@typescript-eslint/parser": "^4.31.0",
 		"eslint": "^7.32.0",
 		"eslint-config-prettier": "^8.3.0",
 		"eslint-plugin-prettier": "^4.0.0",
 		"eslint-plugin-react": "^7.25.1",
 		"husky": "^7.0.2",
+		"jest-expo": "^42.1.0",
 		"prettier": "^2.3.2",
+		"react-test-renderer": "^17.0.2",
 		"typescript": "~4.0.0"
 	},
+	"jest": {
+		"projects": [
+			{
+				"preset": "jest-expo/ios"
+			},
+			{
+				"preset": "jest-expo/android"
+			}
+		],
+		"transformIgnorePatterns": [
+			"node_modules/(?!(jest-)?react-native|react-clone-referenced-element|@react-native-community|expo(nent)?|@expo(nent)?/.*|react-navigation|@react-navigation/.*|@unimodules/.*|unimodules|sentry-expo|native-base|@sentry/.*)"
+		],
+		"testEnvironment": "node"
+	},
```

Comment:
I'd prefer to put this config in a separate `jest.config.js`. Just makes it a little easier to work with

Reply:
can do